### PR TITLE
Updated Resource Quota dashboard layout

### PR DESCRIPTION
**Pull request by dtaylor113**
- Moved 'scopes' to title
- Added 'No Limit/Request' Quota chart placeholders
- Charts now 100% of row
- Removed '--compact' and other unnecessary css selectors

image

JIRA: https://jira.coreos.com/browse/CONSOLE-1257

### CODE REVIEW

File: frontend/public/components/_quota.scss [hunk 1]
```diff
@@ -21,41 +18,16 @@
   padding-left: 34px;
 }
 
-.co-resource-quota-scope__description {
-  font-size: $font-size-small;
+.quota-dashboard-scopes {
```
Comment (rhamilto) [inline frontend/public/components/_quota.scss]: Add `co-` so the classes are consistent?

File: frontend/public/components/overview/namespace-overview.tsx [hunk 2]
```diff
@@ -46,27 +46,23 @@ const OverviewNamespaceSummary = ({ns}) => <div className="group">
...
-      </div>
+  return <div className="group">
+    <div className="group__title">
+      <h2 className="h3">
+        <ResourceLink kind="ResourceQuota" name={quota.metadata.name} className="co-resource-link-truncate"
+          namespace={quota.metadata.namespace} inline="true" title={quota.metadata.name} />
+        {scopes && <QuotaScopesInline className="quota-dashboard-scopes" scopes={scopes} />}
+      </h2>
+    </div>
+    <div className="container-fluid group__body group__graphs">
```
Comment (rhamilto) [inline frontend/public/components/overview/namespace-overview.tsx]: `container-fluid` is not necessary.

File: frontend/public/components/_quota.scss [hunk 3]
```diff
@@ -1,12 +1,9 @@
...
   display: flex;
   flex-wrap: wrap;
-  justify-content: center;
-  margin-bottom: 20px;
-}
-
-.co-resource-quota-chart-row--compact {
-  margin-bottom: 2px;
+  justify-content: space-evenly;
+  margin-left: 40px;
```
Comment (rhamilto) [inline frontend/public/components/_quota.scss]: I'd be inclined to drop both these margin additions as they result in premature wrapping that's not necessarily desirable.

Comment (dtaylor113) [reply frontend/public/components/_quota.scss]: fixed

File: frontend/public/components/overview/namespace-overview.tsx (hunk 2 above)
Comment (dtaylor113) [reply frontend/public/components/overview/namespace-overview.tsx]: fixed

**Comment by dtaylor113** [comment]
@rhamilto, I believe I have addressed all your review comments.  Yikes!  Didn't realize I created the dev branch in master as opposed to my fork!  -will delete after this PR merges

File: frontend/public/components/_quota.scss (hunk 1 above)
Comment (dtaylor113) [reply frontend/public/components/_quota.scss]: fixed

File: frontend/public/components/_quota.scss [hunk 4]
```diff
@@ -21,41 +16,16 @@
   padding-left: 34px;
 }
 
-.co-resource-quota-scope__description {
-  font-size: $font-size-small;
+.co-quota-dashboard-scopes {
```
Comment (rhamilto) [inline frontend/public/components/_quota.scss]: Sorry, @dtaylor113.  It occurs to me this classname should also include `resource` (e.g., `.co-quota-resource-dashboard-scopes`) and be alphabetized within the file to improve consistency (move rule to line 7).

Comment (dtaylor113) [reply frontend/public/components/_quota.scss]: no prob., fixed, thanks :-)

**Comment by spadgett** [comment]
olm test flake

/retest

**Comment by spadgett** [comment]
install failed

/retest

**Pull request merged**